utils/DiscordBot.ts

```diff
@@ -25,14 +26,15 @@ export const getReportChannel = (): Discord.TextChannel => getMainGuild().channe
 export const getLoggingChannel = (): Discord.TextChannel => getMainGuild().channels.cache.get(loggingChannelID) as Discord.TextChannel
 export const getBotReviewLogChannel = (): Discord.TextChannel => getReviewGuild().channels.cache.get(botReviewLogChannelID) as Discord.TextChannel
 export const getStatsLoggingChannel = (): Discord.TextChannel => getMainGuild().channels.cache.get(statsLoggingChannelID) as Discord.TextChannel
+export const getOpenBotReviewLogChannel = (): Discord.TextChannel => getReviewGuild().channels.cache.get(openBotReviewLogChannelID) as Discord.TextChannel
```

Comment:
```suggestion
export const getOpenBotReviewLogChannel = (): Discord.TextChannel => getMainGuild().channels.cache.get(openBotReviewLogChannelID) as Discord.TextChannel
```
공개 심사 로그 채널은 메인 서버에 위치.